src/modlunky2/ui/trackers/runstate.py

```diff
@@ -318,6 +346,8 @@ def update_attacked_with(self, layer: Layer, presence_flags: PresenceFlags):
                 if item_type == EntityType.ITEM_EXCALIBUR and self.theme == Theme.ABZU:
                     self.lc_has_swung_excalibur = True
                     self.failed_low_if_not_chain = True
+                    if not self.is_chain:
+                        self.run_label.discard(Label.LOW)
```

Comment:
This needs to retroactively invalidate low if chain is violated later but i'm reading this as if it only invalidates low if you're not currently chain.

Reply:
We invalidate low in `fail_chain()`, so that case does work out.

Reply:
Why is this discarding the label instead of calling `self.fail_low()`? Is there an undesired interaction with setting `is_low_percent = False`